README.md

```diff
@@ -2,17 +2,20 @@
 * [Description](#description) 
 * [Technologies](#technologies) 
 * [Set-Up](#set-up) 
+* [Server Links](#Server-Links) 
 
 # Description
 Website for remodeling company in Houston, Texas
 
 # Technologies
 * javascript
-* flask
+* deno
 * bootstrap
 
 # Set-Up:
-1. Create a python virtual environment (3.9)
-2. pip install -r requirements.txt
-3. export FLASK_APP=app.py && FLASK_DEBUG=1
-4. flask run
\ No newline at end of file
+1. install deno
+2. $deno run --allow-net --allow-read App/server.js local
+
+# Server Links
+* https://prod-app-6mwmhwfa3a-uc.a.run.app/
```

Comment:
could you make these links open in a new tab on click  it could be done if you use <a href=<link-to-server> target="_blank"></a>

make sure you add text inside the element

and make one for prod and dev

Reply:
other than that it looks good!

Reply:
I've added that. I'm trying to test it out and it seems like it's not working. Is it just me or what am I doing wrong?

Reply:
Okay so it seems markdown doesn't support this. My bad I thought it did. I like how it looks right now with the prod and dev server labeled. Just remove the target attribute and it should be good.

Reply:
alright